test/conftest.py

```diff
@@ -1,8 +1,13 @@
-import sys
+import sys,os
 import time
 from datetime import timezone
 from pathlib import Path
-
+try:
+    DESKTOPPATH = Path(__file__).resolve().parent
+except:
+    print("error encontrando ruta de la aplicacione")
+sys.path.append(os.path.join(Path(__file__).resolve().parent, 'inc'))
```

Comment:
I can send a refactoring for this but it is alright.
With Path() you do `Path(__file__) / 'inc'`